ops/tag-service/README.md

```diff
@@ -0,0 +1,21 @@
+# Tag Service
+Tag Service is a Github action which builds new tags and applies them to services in the monorepo.
+It accepts:
+* Service name
+* Bump Amount [major, minor, patch]
+* Prerelease and Finalize-Prerelease (to add/remove `rc` versions)
+
+It can be triggered from the Github Actions panel in the monorepo
+
+# Tag Tool
+Tag Tool is a minimal rewrite of the Tag Service to let operators prepare and commit tags from commandline
+It accepts:
+* Service name
+* Bump Amount [major, minor, patch, prerelease, finalize-prerelease]
+
+Tag Tool is meant to be run locally, and *does not* perform any write operations. Instead, it prints the git commands to console for the operator to use.
```

Comment:
I'd be interested in an extension which would do the tags. I've seen the suggestion (and I kinda like it) to do a dry run by default and have a `--wet` flag to actually run the commands.

Reply:
Printing out the git commands is a great first step. But agree that we could go one step further and soon have it print a confirmation dialog, and then do the tagging & pushing itself. Could also use [GitPython](https://gitpython.readthedocs.io/en/stable/) for all git reading and interaction.

Reply:
the classic amazonian would be `--no-dry-run`, which I like because it's super heavy and obvious.

the only issues with introducing write operations is the opportunity for error, which an operator can catch. For example, there's no branch validation when this tool runs.

I'm up for adding this feature down the road if not having it is a major inconvenience, but for now operators can make this happen via:

`tag-tool.py [...] | grep git | sh`

Which will isolate git commands and feed them to your shell

Reply:
Sounds good as a follow up!